benchmarker/modules/do_cublas.py

```diff
@@ -14,7 +14,7 @@ def run(self):
                 raise Exception("cublas requires one GPU")
         size = " ".join(map(str, self.params['problem']['size']))
         path_binary = os.path.join(os.path.dirname(os.path.realpath(__file__)),
-                                   "problems/cublas/main")
+                                   "problems/gemm/cublas/main")
```

Comment:
is this correct? does this skip cublas now?

Reply:
it is correct, not sure why it should be skipping anything

Reply:
Just checking, just checking! So I was under the impression that this was something which called cublas, but now it's calling gemm... but I guess now gemm includes cblas and cublas, right?

Reply:
yeah I just moved in a subfolder
the do_cublas can only do one problem, so it did not really matter where it was
no it's more like resent/pytorch